scripts/check-delegates.mjs

```diff
@@ -19,8 +19,44 @@ export function checkDelegates(contents = fs.readFileSync('./delegates.txt', 'ut
     'VM', 'WH', 'YK', 'ZB',
   ]);
 
+  // list of abbreviations that are not ideal.
+  // most of these are here because they are grandfathered in.
+  // new elements should only be false positives.
+  const NON_IDEAL_ABBRS = new Set([
```

Comment:
How was this list generated? I would think it would either have no 2-letter abbreviations (preferred) or all 2-letter abbreviations. But instead it only has some? How do we know that each entry is necessary?

Reply:
it was generated by replacing line 100 with pushing to an array to write out all the existing 'non-ideal' entries that appeared

Reply:
Then how do you not have an entry for all 2-letter abbreviations?

Reply:
last week seems like an eternity ago..  it was running on an earlier version of this code, and IIRC, I had originally combined the 2 letter check with the ideal check, but this resulted in messing with determinism and the test assertions, so I separated it out and moved the check to be last.  will update, good catch.

Reply:
fixed